CHANGELOG.md

```diff
@@ -5,10 +5,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [1.0.0-beta.59](https://github.com/Esri/calcite-components/compare/v1.0.0-beta.58...v1.0.0-beta.59) (2021-07-07)
 
+* **calcite-button:** stop spreading attributes (#2410)
```

Comment:
Let's remove this since it's out of place and we can add the scope in the notes below. The issue is linked in the bug fixes section, so we can omit it. I'll leave that up to you.